pkg/providerimages/mirrors_test.go

```
@@ -52,17 +52,39 @@ var _ = Describe("sourceMatchesRef", func() {
+		mirrors       map[string][]string
+		expected      string
+		expectedSource string
 	}
 
 	DescribeTable("should resolve image references through mirrors",
 		func(tt testInput) {
-			result := resolveImageRef(tt.imageRef, tt.mirrors)
-			Expect(result).To(Equal(tt.expected))
+			resolved, matchedSource := resolveImageRef(tt.imageRef, tt.mirrors)
+			Expect(resolved).To(Equal(tt.expected))
+			if tt.expectedSource != "" {
```

golangci-lint (2.11.4)

[error] missing whitespace above this line (invalid statement above if)

(wsl_v5)